docs_issues.qmd

```diff
@@ -64,3 +64,31 @@ When using 'devtools', the function [`devtools::document()`](https://www.rdocume
 :::
 
 ---------------------------------------------------------
+
+# Overall Checktime
+
+## Problem
+
+Automatic CRAN tests give a NOTE saying the overall checktime of your package is too long.
+
+## Solution 
+
+Reduce the length and quantity of your examples, vignettes and tests.
+
+### Details
+
+::: {.callout-note title="CRAN NOTE" icon=false}
+
+Check: Overall checktime, Result: NOTE\
+&emsp; Overall checktime 20 min > 10 min
+
+  
+:::
+
+
+Unwrapped examples, vignettes and tests of packages are run frequently on CRAN's test servers. The overall time it takes to run all of them should therefore be less than 10 minutes in total. 
+
+This should not discourage from writing extensive test and documentation with high code coverage but rather encourage sensible testing. Avoid vignettes or tests with many iterations, run them on small toy data or provide precomputed results. 
```

Comment:
It is know that CRAN checks has led to many maintainers not checking all things on CRAN (not only because of time but also due to internet connections).   

I would frame it starting from a positive stance: "Write vignettes and tests with few iterations, run them on small toy data, focus and make sensible tests instead of disabling tests or removing examples."

Reply:
That's a good point, I will rephrase that a bit.

About the internet connections: There is already a discussion going on (#65) and the recipe will follow soon!